worker/worker.go

```diff
@@ -401,6 +402,11 @@ func (w *Worker) parseAndSaveIssues(issueData issueQueryWithBefore, repository *
 		fmt.Println(errors.WithMessage(err, "failed to update issues"))
 	}
 
+	cacheConn := cache.CachePool.Get()
+	defer cacheConn.Close()
+
+	cache.DeleteAllKeys(&cacheConn)
```

Comment:
Deleting everything might break things down the road.

 Could you try prefixing the cache keys (for example for pages you prefix the keys with `page_` and for count with `count_`) and instead of deleting everything only delete the keys that start `page_` and the keys that start with `count_`

Reply:
currently, the keys are prefixed with issues: and issues-count

Reply:
You are right, my bad. Could you delete only the keys that have these prefixes instead of all the keys?

Reply:
Sure! with some scan... since there is no direct command for such an operation i guess.

Reply:
Yeah, the suggested way (if I remember correctly) is to scan for the keys and then use the unlink command to drop all the keys that were returned from the scan